lighthouse-core/test/audits/display-test.js

```diff
@@ -66,4 +66,28 @@ describe('Mobile-friendly: display audit', () => {
   it('succeeds when a complete manifest contains a display property', () => {
     return assert.equal(Audit.audit({Manifest: exampleManifest}).rawValue, true);
   });
+
+  it('has no debugString when successful', () => {
```

Comment:
I think it would be better to just inline a test for `debugString` to exist or not in the above tests instead of adding new tests just for `debugString`